README.md

```diff
@@ -106,8 +106,9 @@ from rio_tiler_pds.sentinel.aws import S1L1CReader
 from rio_tiler_pds.sentinel.aws import (
     S2L1CReader,  # JPEG2000
     S2L2AReader,  # JPEG2000
-    S2L2ACOGReader,   # COG
```

Comment:
Do we know for certain that L1C won't be converted to COG in the future too? It seems more future-proof to leave as `S2L2ACOGReader`, but for now you could alias `S2COGReader` to `S2L2ACOGReader`

Reply:
@kylebarron I don't know if L1C will be available one day but the `S2COGReader` would be working for both L1C and L2A anyway, We can make an alias for S2L2ACOGReader (pointing to `S2COGReader`) ... but it's seems a bit verbose. 

The other readers (L1C, L2A for jpeg2000) are needed because the way the archive is. Maybe we could create an alias for S2JP2Reader which could redirect to either `S2L1CReader` or `S2L2AReader`

Reply:
> The other readers (L1C, L2A for jpeg2000) are needed because the way the archive is

Well that's kinda my point... how do we know a possible future L1C COG archive would be set up the same way as the current L2A COG archive? If we want to minimize the likelihood of future breaking changes, it seems like keeping `S2L2ACOGReader` makes sense, but that an alias could make sense for now as well